#  Message describing a predicted trajectory that contains an array of stamped locations

uint64                          id          # ID of the tracked person
geometry_msgs/Vector3Stamped[]  trajectory
